servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/Karaf2xPublishController.java

```diff
@@ -213,12 +209,20 @@ public int transferBuiltModule(IModule[] module, IPath srcFile, IProgressMonitor
 
 	@Override
 	public int removeModule(IModule[] module, IProgressMonitor monitor) throws CoreException {
-		BundleDetails bd = new ModuleBundleVersionUtility().getBundleDetails(module, null);  // This will clearly break... wtf.  Crap. 
+		IPath tmpArchive = getTempBundlePath(module);
+		
+		BundleDetails bd = new ModuleBundleVersionUtility().getBundleDetails(module, tmpArchive);  // This will clearly break... wtf.  Crap. 
```

Comment:
If the temporary archive doesn't exist, and it's an osgi bundle rather than a fuse, it will fail to read the information from the osgi file and this will fail.